autora/cycle/plot_utils.py

```diff
@@ -472,3 +480,102 @@ def plot_results_panel_3d(
     )
 
     return fig
+
+
+def cycle_default_score(cycle: Cycle, x_vals: np.ndarray, y_true: np.ndarray):
+    """
+    Calculates score for each cycle using the estimator's default scorer.
+    Args:
+        cycle: AER Cycle object that has been run
+        x_vals: Test dataset independent values
+        y_true: Test dataset dependent values
+
+    Returns:
+        List of scores by cycle
+    """
+    l_scores = [s.score(x_vals, y_true) for s in cycle.data.theories]
+    return l_scores
+
+
+def cycle_specified_score(
+    scorer: Callable, cycle: Cycle, x_vals: np.ndarray, y_true: np.ndarray, **kwargs
+):
+    """
+    Calculates score for each cycle using specified sklearn scoring function.
+    Args:
+        scorer: sklearn scoring function
+        cycle: AER Cycle object that has been run
+        x_vals: Test dataset independent values
+        y_true: Test dataset dependent values
+        **kwargs: Keyword arguments to send to scoring function
+
+    Returns:
+
+    """
+    # Get predictions
+    if "y_pred" in inspect.signature(scorer).parameters.keys():
+        d_y_pred = _theory_predict(cycle, x_vals, predict_proba=False)
+    elif "y_score" in inspect.signature(scorer).parameters.keys():
+        d_y_pred = _theory_predict(cycle, x_vals, predict_proba=True)
+
+    # Score each cycle
+    l_scores = []
+    for i, y_pred in d_y_pred.items():
+        l_scores.append(scorer(y_true, y_pred, **kwargs))
+
+    return l_scores
+
+
+def plot_cycle_score(
+    cycle: Cycle,
+    X: np.ndarray,
+    y_true: np.ndarray,
+    scorer: Optional[Callable] = None,
+    x_label: str = "Cycle",
+    y_label: Optional[str] = None,
+    figsize: Tuple[float, float] = (6.4, 4.8),
```

Comment:
Is there a way we can just inherit this from matplotlib? You could have some logic like: "if the user passes a figure themselves, then use that. If there's no figure passed in, then make one using the default parameters from matplotlib."

Reply:
Updated the script to access Matplotlib's rcParams to get the default figure size.